internal/controller/management/projects/projects.go

```diff
@@ -926,3 +1005,10 @@ func (r *reconciler) migrateSpecToProjectConfig(
 func getRoleBindingName(serviceAccountName string) string {
 	return fmt.Sprintf("%s-read-secrets", serviceAccountName)
 }
+
+func generateWebhookPath(project, provider, secret string) string {
+	input := []byte(project + provider + secret)
+	h := sha256.New()
+	h.Write(input)
+	return fmt.Sprintf("/%x", h.Sum(nil))
+}
```

Comment:
It might be better to make this a method on the `ReceiverConfig`, especially once we determine what Secret value we want to use. Instead of calling it a Secret, which is quite an overloaded term in the Kubernetes landscape, I would call it a "token" (value).